### ISSUE TITLE
Documentation Table of Contents not visible during tab focus

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1203]
Auditor issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/60646e32-5884-11ee-be02-0f9218df4b24?sortField=ordinal&sortDir=asc&filter%5Brule%5D=manual&filter%5Bpage_number%5D=49&row=0

The element is not visible when it receives focus by default.
Element description: Below links.
ComboboxGroup
Related Components

Element location: In the section "On this page"

Additional Information: When the keyboard user is navigating through the page using tab key navigation, the elements are focused but are not visible to the keyboard users.
However when the user scrolls the page using a down arrow key, the keyboard user is able to see those links, hence raised as a best practice.

## Dev Notes

This appears to be an issue in shorter viewports with regards to the Table of Contents links on the right area of documentation. We should consider making this area scrollable when the on this page links do not fit.
